Custo eletronica

kewinkiister opened the pull request:
Foi feito a tabela de custos para eletrônica

kewinkiister committed:
Alteração

kewinkiister committed:
Alteração

kewinkiister committed:
Alterações

kewinkiister committed:
Adicao da tabela de custos de eletrônica

marimendes commented on editaveis/aspectosgerais.tex:
```diff
@@ -1,18 +1,100 @@
+\end{table}
 \section{Viabilidades financeira}
 \section{Levantamento de riscos}
+<<<<<<< HEAD
```
Ta com um conflito aqui

kewinkiister replied:
Corrigi e subi a nova versão já

kewinkiister replied:
O que tenho que fazer agora?

marimendes reviewed (changes requested):
Tem um conflito pra resolver

kewinkiister committed:
Correção de conflito

marimendes commented on editaveis/aspectosgerais.tex:
```diff
@@ -1,18 +1,98 @@
+
+\end{comment}
+
+>>>>>>> 4ef4c5a9d179b0ced8d6ff52122a2aacd641c19d
```
Ficou faltando apagar isso aqui só.

kewinkiister replied:
Tá vê agora de novo

kewinkiister committed:
Correção de conflito

sconetto reviewed (changes requested):
Os arquivos `.DS_Store`, `rudix.py`, `tcc.bcf` e `tcc.run.xml` estão no PR e não deveriam ser incluídos no repositório.
Nesse PR tem também adições da branch de EAP e da branch de introdução, está correto?
Tem adição no README que não é necessária (adiciona um "a").
Na linha  59 do `aspectosgerais.tex` começa um comentário, não seria o caso de remover essa parte do texto? (o mesmo vale para o `elementosdopostexto.tex`)

kewinkiister commented:
Eu posso dar o comando (git rm meu_arquivo.txt) para excluir essas branchs que eu baixei?

sconetto committed:
Limpeza de arquivos

Remove arquivos indesejáveis e arruma algumas inserções desnecessárias
ao projeto

Signed-off-by: João Pedro Sconetto <sconetto.joao@gmail.com>

sconetto committed:
Corrige escrita e remove comentários

Última limpeza da branch para o merge

Signed-off-by: João Pedro Sconetto <sconetto.joao@gmail.com>

sconetto committed:
Merge branch 'develop' into custo_eletronica
